### PR TITLE
16 fix not require field cant store with null value

### LINKED ISSUE #16: fix not require field can't store with null value

### PR DESCRIPTION
**Pull request by FishChen1202**

### CODE REVIEW

**Comment by FishChen1202** [comment]
[In custom data, the values for rpURL, typURL, and referrer are empty, causing the package to block.](https://trello.com/c/nwkqrBb6/1837-in-custom-data-the-values-for-rpurl-typurl-and-referrer-are-empty-causing-the-package-to-block)

File: database/migrations/2024_01_09_028373_change_value_column_of_custom_filed_values_table_to_nullable.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('custom_field_values', function (Blueprint $table) {
+            $table->text('value')->nullable()->change();
```
Comment (chenghung) [inline database/migrations/2024_01_09_028373_change_value_column_of_custom_filed_values_table_to_nullable.php]: :question: 會想變成nullable是想要存custom value是null值？

或是讓它繼續保持只存有值的custom values
從程式去處理, 在保存之前先排除掉所有null值

**Review by chenghung — COMMENTED** [review]
@FishChen1202 comments.

**Review by jh0712 — APPROVED** [review]
r+

**Pull request merged**